src/content/general/architcture/openstack/index.md

```diff
@@ -0,0 +1,102 @@
+---
+linkTitle: Openstack
+title: The Giant Swarm Openstack architecture
+description: Architecture overview showing how Giant Swarm is set up within a customer data center on Open stack deployment, using cluster API.
+weight: 30
+menu:
+  main:
+    parent: general-architecture
+last_review_date: 2022-02-02
+user_questions:
+  - Do you run Openstack?
+aliases:
+  - /basics/openstack-architecture/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+
+
+# MC bootstraping
```

Comment:
The main headline should be removed, as it is now part of the template and populated from the `title` front matter field.